src/main/java/meteordevelopment/meteorclient/systems/modules/combat/Offhand.java

```diff
@@ -147,20 +204,22 @@ private boolean usableItem() {
 
     @Override
     public String getInfoString() {
-        return item.get().name();
+        return preferredItem.get().name();
     }
 
     public enum Item {
-        EGap(Items.ENCHANTED_GOLDEN_APPLE),
-        Gap(Items.GOLDEN_APPLE),
+        eGap(Items.ENCHANTED_GOLDEN_APPLE),
+        gap(Items.GOLDEN_APPLE),
```

Comment:
why

Reply:
camelCasing

Reply:
You shouldn't really use camel case for an enum

Reply:
Fixed, Ty for the feedback